src/main/java/net/neoforged/moddevgradle/dsl/ModDevExtension.java

```diff
@@ -158,4 +158,29 @@ public void ideSyncTask(Task task) {
     public void addModdingDependenciesTo(SourceSet sourceSet) {
         ModDevArtifactsWorkflow.get(project).addToSourceSet(sourceSet);
     }
+
+    /**
+     * After enabling modding, you can retrieve the version of the modding platform you picked using this getter.
+     * I.e. the NeoForge or Forge version. If you chose to enable vanilla-only mode, this getter returns null.
+     */
+    @Nullable
```

Comment:
That's just weird I think. Better to throw if the NeoForge version was not specified, IMO.